merlin/server/server_commands.py

```diff
@@ -243,6 +272,28 @@ def start_server() -> bool:  # pylint: disable=R0911
 
     LOG.info(f"Server started with PID {str(process.pid)}.")
     LOG.info(f'Merlin server operating on "{redis_out["hostname"]}" and port "{redis_out["port"]}".')
+    return True
+
+
+def start_server() -> bool:  # pylint: disable=R0911
+    """
+    Start a merlin server container using singularity.
+    :return:: True if server was successful started and False if failed.
+    """
+    if not check_for_not_running_server():
+        return False
+
+    server_config = pull_server_config()
+    if not server_config:
+        LOG.error('Try to run "merlin server init" again to reinitialize values.')
+        return False
+
+    process = start_redis_container(server_config)
```

Comment:
Ok good call. I think I can just change this to `start_container` instead. I'm looking at the logic of `start_redis_container` and it still seems generalized enough that this would be acceptable.